app/views/api/v2/remote_execution_features/interface_extensions.rabl

```diff
@@ -0,0 +1 @@
+attributes :execution
```

Comment:
Rabl is just a templating language, the template files usually include the "resulting" format after rendering, so for api the filename should end with `.json.rabl`

Reply:
Changed, thanks!